web/pgadmin/tools/sqleditor/__init__.py

```diff
@@ -953,6 +953,15 @@ def poll(trans_id):
                 if columns_info is not None:
                     # Only QueryToolCommand or TableCommand can be editable
                     if hasattr(trans_obj, 'obj_id') and trans_obj.can_edit():
+                        st, res = fetch_sequences_data(trans_obj)
```

Comment:
Why do we need separate SQL query for sequence type? This should be taken care in `get_columns_types` and in the one SQL.

Reply:
I have made changes in the function you have mentioned and it's working, I just modified the nodes.sql to fetch the sequences data.

Reply:
I cannot see the latest code somehow. Did you commit?

Reply:
I haven't, will commit it with front end changes. If you want I can commit that now.